packages/objection/src/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { Model } from 'objection';
+
+function posixRegex(ignoreCase: boolean) {
+  const operator = ignoreCase ? '~*' : '~';
+  return `:field: ${operator} :regex`;
+}
+
+function regexp() {
+  return ':field: regexp :regex = 1';
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/regexp.html#function_regexp-like Supports mathing type, so we can use ignore case for MySQL as well

Reply:
regexp_like is not supported in mysql 5.7. Are you ok with that?

Reply:
Ah, I see. Thanks for clarification.